src/App/examples/Update-AzContainerApp.md

```diff
@@ -1,6 +1,11 @@
 ### Example 1: Update a Container App.
 ```powershell
-Update-AzContainerApp -Name azps-containerapp -ResourceGroupName azpstest_gp -Location canadacentral -DaprEnabled -DaprAppProtocol 'http' -DaprAppId "container-app-1" -DaprAppPort 8080
+$secretObject = Get-AzContainerAppSecret -ContainerAppName azps-containerapp -ResourceGroupName azpstest_gp
+$newSecretObject = @(0..($secretObject.Count-1))
+[array]::copy($secretObject,$newSecretObject,$secretObject.Count)
+$secretObject += New-AzContainerAppSecretsecretObject -Name "yourkey" -Value "yourvalue"
```

Comment:
The analysis is correct, please correct New-AzContainerAppSecretsecretObject -> New-AzContainerAppSecretObject

Reply:
Please correct this typo in generation branch as well